ncai-developers/check-mate
# added api server

to test:
 make sure you have yarn installed globally
```
npm install -g yarn
```

then run
```
yarn install
```
to kick off the server
```
yarn run api
```
navigate to `localhost:3000/api/017770` and you should see some json object
